tests/test_app.py

```diff
@@ -134,12 +138,13 @@ def test_title_search_success(self, mock_get):
         assert 'Luton' in page_content
         assert 'LU1 1DZ' in page_content
 
-    def test_title_search_invalid_search_value_format(self):
+    @mock.patch('requests.get', return_value=fake_no_titles)
```

Comment:
By analogy, we could check if the title appears in the search result if contained in the API response.